sdk/ai/ai-projects/src/agents/files.ts

```diff
@@ -22,39 +29,41 @@ enum FilePurpose {
 /** Gets a list of previously uploaded files. */
 export async function listFiles(
   context: Client,
-  options?: ListFilesParameters,
+  options: ListFilesOptionalParams = {},
 ): Promise<FileListResponseOutput> {
-  validateListFilesParameters(options);
+  const listOptions: GeneratedParameters.ListFilesParameters = {
+    ...operationOptionsToRequestParameters(options),
+  };
+  validateListFilesParameters(listOptions);
   const result = await context.path("/files").get(options);
   if (!expectedStatuses.includes(result.status)) {
       throw createRestError(result);
   }
-  return result.body; 
+  return ConvertFromWire.convertFileListResponseOutput(result.body); 
 }
 
 /** Uploads a file for use by other operations. */
 export async function uploadFile(
   context: Client,
-  options: UploadFileParameters,
+  options: GeneratedParameters.UploadFileParameters,
```

Comment:
This is using GeneratedParameters, it should be from Cutomization

Reply:
Updated the parameters to be from customization